definitions/info_object_meta.toml

```diff
@@ -22,6 +22,18 @@ type = "time"
 type = "string"
 description = "LinkTarget is the symlink target for link object."
 
+[max-part-count]
```

Comment:
How about:

- `multipart-number-maximum`
- `multipart-size-maximum`
- `multipart-size-minimum`

Keep the `multipart` prefix so that we know those value only provides for part object.